app/src/pages/inside/dashboardPage/modals/addEditModal/addEditModal.scss

```diff
@@ -47,3 +47,7 @@
     }
   }
 }
+
+.share-switcher-wrapper > div > label {
```

Comment:
What if component's layout will change? I suggest changing `BigSwitcher` root element style from `display: flex` to `display:inline-flex`

Reply:
I'm afraid that this could affect some other places where BigSwitcher is used

Reply:
It's better to fix the root cause of a problem than try to fix many similar local bugs